index.html

```diff
@@ -42,7 +42,10 @@ <h2>Danger Zone</h2>
         <div class="content is-small">
             <p>Data is <span class="is-italic">not</span> transferred between machines but <span class="is-italic">is</span> preserved when closing and re-opening the site in the same browser.</p>
             <p><a href="https://github.com/anthonyisensee/incidenttracker/issues/new?labels=bug">Report an issue</a> or <a href="https://github.com/anthonyisensee/incidenttracker/issues/new?labels=enhancement">submit a feature request</a>. Feel free to fork <a href="https://github.com/anthonyisensee/incidenttracker">the code</a> and submit a <a href="https://github.com/anthonyisensee/incidenttracker/pulls">pull request</a>.</p>
-            <p>Developed by <a href="https://github.com/anthonyisensee">Anthony Isensee</a> for Ricky Moore. 🖤</p>
+            <p>
+                Developed by <a href="https://github.com/anthonyisensee">Anthony Isensee</a> for Ricky Moore. 🖤
+                Thank you to <a href="https://github.com/geo-bold">Geo Archbold</a> for contributions.
```

Comment:
Truly, thank you!! I've had a ton of fun with this.

Reply:
It has been an absolute pleasure! Thank you!